recipes/gwyfile/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "gwyfile" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cf72fedd6af15c952067dbbe87978a4a6262b4e2cd3aa9f2e7bee4ced4071e40
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - versioneer
```

Comment:
Is `versioneer` needed as a dependency? Or does the `setup.py` use the copy that's in the top level of the tarball?

Reply:
yup, I guess you are right with the latter!